6. Developer Guides/09. Realtime-API/1. Method Calls/4 . Load History/README.md

```diff
@@ -0,0 +1,40 @@
+# Load History
+
+Use this method to make the initial load of a room. After the initial load you may subscribe to the room messages stream (see [Stream Room Message][1]).
+
+This method accepts 4 parameters in the following order:
+- The room id
+- (?)
```

Comment:
@laggedHero If the second parameter is passed, which is a date, then the method will find messages which are older than the passed in time and return them as the result's message property. If the `ls`, aka last seen` is passed in, then there will be `firstUnread` (message object) and `unreadNotLoaded` (number) properties on the result object.

Does that make sense?